### DISCUSSION TITLE
Does LinearMouse have to be running in the background to disable mouse acceleration?

### DISCUSSION THREAD

**Discussion by harrisonjr98** [linearmouse/linearmouse#269] [Q&A]
Looking for a solution to disable mouse acceleration on my macbook, but *only* when using it docked with a USB mouse. This looks to be the ticket, but does it require the app to be running at all times in the background to accomplish this? If so, is it recommended to add it as a Login Item for uninterrupted operation? Thanks!

**Comment by lujjjh** [comment]
Hi, @harrisonjr98!

> Does LinearMouse have to be running in the background to disable mouse acceleration?

Yes. LinearMouse will disable the acceleration for the mouse each time it is reconnected to the MacBook. Otherwise, you will have to manually launch LinearMouse after plugging in the mouse and after rebooting the computer.

> If so, is it recommended to add it as a Login Item for uninterrupted operation?

Please refer to #260.
